### PR TITLE
databe added

### PR DESCRIPTION
**Pull request by AhsanBracu**

### CODE REVIEW

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java [hunk 1]
```diff
@@ -0,0 +1,781 @@
...
+import com.framgia.hrm.model.Position;
+import com.framgia.hrm.model.SearchStaff;
+import com.framgia.hrm.model.Staff;
+import com.framgia.hrm.model.Status;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: remove this blank line please

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java [hunk 2]
```diff
@@ -0,0 +1,781 @@
...
+    private static final String FTS_TABLE_CREATE =
+            "CREATE VIRTUAL TABLE " + FTS_VIRTUAL_TABLE +
+                    " USING fts3 (" +
+                    COLUMN_NAME + ", " +
+                    FTS_BIRTH_DATE_FIELD + ", " +
+                    FTS_BIRTH_PLACE_FIELD + ", " +
+                    COLUMN_PHONE + ", " +
+                    SEARCH_STAFF +
+                    ")";
+
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: Blank line!

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java [hunk 3]
```diff
@@ -0,0 +1,781 @@
...
+    private static final String STAFF_COLUMN_ID = "staff_id";
+    private static final String STAFF_COLUMN_NAME = "name";
+    private static final String STAFF_COLUMN_DATE_OF_BIRTH = "date_of_birth";
+    private static final String STAFF_COLUMN_BIRTH_PLACE = "birth_place";
+    private static final String STAFF_COLUMN_PHONE_NUMBER = "phone_number";
+    private static final String STAFF_COLUMN_DEPT = DEPARTMENT_COLUMN_ID;
+    private static final String STAFF_COLUMN_STATUS = STATUS_COLUMN_ID;
+    private static final String STAFF_COLUMN_ACTIVITY = ACTIVITY_COLUMN_ID;
+    private static final String STAFF_COLUMN_POSITION = POSITION_COLUMN_ID;
+
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: Blank line

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java [hunk 4]
```diff
@@ -0,0 +1,781 @@
...
+
+    /**
+     * department table create statement
+     */
+    private static final String CREATE_TABLE_DEPARTMENT = "CREATE TABLE " + TABLE_DEPARTMENT
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: if table existed???

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java [hunk 5]
```diff
@@ -0,0 +1,781 @@
...
+        cursor.close();
+        db.close();
+
+        return id;
+
+    }
+    /**
+     * Getting staff by id
+     */
+   /* public Staff getStaffById(int staff_id) {
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: remove comment out method

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java [hunk 6]
```diff
@@ -0,0 +1,781 @@
...
+                staff.setActivity_id(c.getInt(c.getColumnIndex(STAFF_COLUMN_ACTIVITY)));
+                staff.setPosition_id(c.getInt(c.getColumnIndex(STAFF_COLUMN_POSITION)));
+
+                staffs.add(staff);
+            } while (c.moveToNext());
+        }
+
+        return staffs;
+    }
+    /*public ArrayList getStaffByDepartment(long dept_id) {
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: remove comment out method

File: app/src/main/res/layout/content_main.xml [hunk 7]
```diff
@@ -8,25 +8,30 @@
...
+            android:id="@+id/list_department"
             android:layout_width="fill_parent"
-            android:layout_height="@dimen/content_main_frame_container_body_layout_height"
-            android:layout_weight="1" />
-    </LinearLayout>
+            android:layout_height="fill_parent"
+            android:transcriptMode="alwaysScroll"
+            android:cacheColorHint="#00000000"
+            android:listSelector="@android:color/transparent"/>
+
```
Comment (nvmanh) [inline app/src/main/res/layout/content_main.xml]: blank line

File: app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java (hunk 4 above)
Comment (AhsanBracu) [inline app/src/main/java/com/framgia/hrm/database/DatabaseHelper.java]: used sharedpreference

**Pull request merged**